### PR TITLE
Ddcnl 5409

### PR DESCRIPTION
**Pull request by Sachin300793**

### CODE REVIEW

File: app/config/ConfigDecorator.scala [hunk 1]
```diff
@@ -267,6 +267,10 @@ class ConfigDecorator @Inject() (
   lazy val sessionTimeoutInMinutes = sessionTimeoutInSeconds / 60
   lazy val sessionCountdownInSeconds = runModeConfiguration.getOptional[Int]("ptaSession.countdown").getOrElse(120)
 
+  lazy val incomeTaxViewChangeFrtonendHost = getExternalUrl(s"income-tax-view-change-frontend.host").getOrElse("")
+
+  lazy val itsaViewUrl = s"$incomeTaxViewChangeFrtonendHost/report-quarterly/income-and-expenses/view"
```
Comment (pascal22p) [inline app/config/ConfigDecorator.scala:272]: The origin is missing on the itsa url:
/report-quarterly/income-and-expenses/view?origin=PTA

File: conf/messages [hunk 2]
```diff
@@ -448,7 +448,7 @@ label.make_a_payment=Make a payment
 label.view_your_payments=View your payments
 label.check_if_you_need_to_fill_in_a_tax_return=Check if you need to fill in a tax return
 label.find_out_how_to_access_self_assessment=Find out how to access Self Assessment
-label.view_and_manage_your_self_assessment_tax_return_the_deadline_for_online_=View and manage your Self Assessment tax return. The deadline for online returns is 31 January {0}.
+label.view_and_manage_your_self_assessment_tax_return_the_deadline_for_online_=View and manage your Self Assessment tax return.
```
Comment (pascal22p) [inline conf/messages:451]: The old label should be kept for the old sa tile.

**Review by pascal22p — CHANGES_REQUESTED** [review]

File: conf/messages (hunk 2 above)
Comment (SandeepUmredkar) [reply conf/messages:451]: We have broken down the label in two parts. The label still remains intact. But first part being used in multiple places hence we did this. I think we forgot to update the old SA tile to reflect using 2 keys. will sort it out.

File: app/config/ConfigDecorator.scala (hunk 1 above)
Comment (Sachin300793) [reply app/config/ConfigDecorator.scala]: Done

**Pull request closed without merging**